src/app/ad-authoring/ad-authoring.ts

```diff
@@ -0,0 +1,5 @@
+export interface AdAuthoring {
```

Comment:
I don't think having a separate interface is necessary for these fields, they can be declared directly in the state.